docs/index.md

```diff
@@ -81,12 +91,49 @@ to be named "my-commands" you'd put the following in your .bashrc or .zshrc:
 
 ```
 # posix example, e.g. .bashrc
-eval "`~/bin/tome init my-commands ~/my-scripts $0`"
+eval "$(tome init my-commands ~/my-scripts bash)"
+
+# zsh
+eval "$(tome init my-commands ~/my-scripts zsh)"
+```
+
+For fish shell:
+```
+# in ~/.config/fish/conf.d/tome.fish
+tome init my-commands ~/my_script_dir $0 | source
 ```
 
 *NOTE*: make sure to include the double quotes with the nested backticks. This ensures that newlines are captured
 and evaluated appropriately.
 
 ### 4. Start a New Shell
 
-Once installed, start a new shell and you should have your new command!
\ No newline at end of file
+Once installed, start a new shell and you should have your new command!
+
+## Developing Tome
```

Comment:
might consider moving this to a `CONTRIBUTING.md`.

Reply:
Good idea, will do

Reply:
Done!